doc/yay.8

```diff
@@ -532,6 +552,16 @@ Shows statistics for installed packages and system health.
 pacman -Qmq | grep -Ee '-(cvs|svn|git|hg|bzr|darcs)$' | yay -S --needed -
 pacaur-like devel check.
 
+.SH ENVIRONMENT VARIABLES
+.TP
+.B AURDEST
+Can be set to configure the builddir.
```

Comment:
```suggestion
Can be set to configure the build directory
Overrides --builddir.
```
Just a mention that it overrides the builddir would be nice

Reply:
It doesn't override `--builddir`. The priority is `flags > environment > config`. I said `builddir` to match the flag name. But maybe I'll take this suggestion and change it to `Overridden by --builddir`.

Reply:
Fair enough, my confusion, just something in terms of priorities so it can be pointed to in order to shame users when they complain that their setting in `.config` didn't work :innocent: